net-news/yencode/yencode-0.46-r1.ebuild

```diff
@@ -1,8 +1,10 @@
-# Copyright 1999-2009 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
+EAPI=6
```

Comment:
Those are major changes, so please drop it to `~arch` and keep the old version until it's stabilized.